### ISSUE TITLE
Errors when converting textures

### ISSUE THREAD

**Issue by Ksandr0** [enenra/space-engineers-utilities#327]
**Describe the bug**
I'm trying to install the module. I set the path to the game folder. I created a folder for assets and specified it in the settings (C:\seassets ). When I press the "Update Textures from Game Files" button, errors appear in the console.

Example:
ERROR - C:\seassets\Textures\Particles\Smoke_SpriteSheet_A_90f_Shrink.TIF
b'\x91\xa8\xe1\xe2\xa5\xac\xa5 \xad\xa5 \xe3\xa4\xa0\xa5\xe2\xe1\xef \xad\xa0\xa9\xe2\xa8 \xe3\xaa\xa0\xa7\xa0\xad\xad\xeb\xa9 \xaf\xe3\xe2\xec.\r\n'

That:
"Системе не удается найти указанный путь." (google translate: "The system cannot find the specified path.")

**To Reproduce**
1. Go to Edit --> Preferences... --> Add-ons
2. Click on expand menu module
3. Set gamedir, assets dir, havok dir, install additions.
4. Open console
5. Click  "Update Textures from Game Files" : https://imgur.com/0enmSD4.png
6. Errors in console: https://imgur.com/3FRNw6N.png

**Expected behavior**
I expect the textures import to pass without error.

**Screenshots**
https://imgur.com/0enmSD4.png
https://imgur.com/3FRNw6N.png

**Versions (please complete the following information):**
 - SEUT version 0.9.96
 - Blender version: 3.1.2

**Additional context**
Windows 11

**Comment by enenra** [comment]
Hi, thanks for the report!

I'm honestly not quite sure what to make of this. Regarding this: 
> "Системе не удается найти указанный путь." (google translate: "The system cannot find the specified path.")
Do you mean that the "\xe3" etc *translate* to that cyrillic phrase? Or that that cyrillic phrase is shown elsewhere?

**Comment by Ksandr0** [comment]
Sorry for delay.

I tried it with 1.0.0-beta.5, the problem is still there. 

There are no listed files in the target folder - C:\seassets\Textures\Particles\Smoke_SpriteSheet_A_90f_Shrink.TIF, for example.

There are files in the game folder, in the "Content" subdirectory - C:\Program Files (x86)\Steam\steamapps\common\SpaceEngineers\Content\Textures\Particles\Smoke_SpriteSheet_A_90f_Shrink.dds

**Comment by enenra** [comment]
Unfortunately I have no idea how to go about fixing it since I can't reproduce it on my end.

The only thing I can think of atm is to circumvent the entire thing by using the old BulkTexture converter that was used before I integrated the functionality into SEUT itself: https://github.com/AtlasTheProto/SETextureConverter

**Comment by enenra** [comment]
With no further activity on this issue for quite a while, I'm going to close it. 

Please let me know if there are any further developments.

**Comment by mugwortmugwort** [comment]
@enenra I had the same issue, using the portable blender versions. I made it print the command to call texconv.exe and it seems in the release version 1.2.0, this was pointing to an invalid location in AppData when calling the blender function bpy.utils.user_resource("SCRIPTS") to find the script folder location. I've just checked in the dev version though and it seems this has been changed now anyway.

If OP @Ksandr0 still needs a workaround they could change the last line of space-engineers-utilities/utils/seut_tool_utils.py:

```
def get_tool_dir() -> str:
    return os.path.join(bpy.utils.user_resource("SCRIPTS"), 'addons', __package__[:__package__.find(".")], 'tools')
```    
to:
```
def get_tool_dir() -> str:
    return os.path.join("C:\\YOUR_PATH_TO_UNZIPPED_BLENDER_VERSION\\4.1\\scripts\\addons", __package__[:__package__.find(".")], 'tools')
```
or possible change it to how it is in the dev branch but I have not tested this for portable installations.
